### PR TITLE
Add gitter to README.md

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Add gitter to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +11,8 @@ View the [latest documentation](http://conveyal-data-tools.readthedocs.org/en/la
 Note: `dev` branch docs can be found [here](http://conveyal-data-tools.readthedocs.org/en/dev/).
 
 ## Docker Image
-The easiest way to get `datatools-server` running is to use the provided `Dockerfile` and `docker-compose.yml`. The `docker-compose.yml` includes both database servers that are needed. Edit the supplied configurations in the `configurations` directory to ensure the server starts correctly. Once this is done running `docker-compose up` will start Datatools and all required database servers.
\ No newline at end of file
+The easiest way to get `datatools-server` running is to use the provided `Dockerfile` and `docker-compose.yml`. The `docker-compose.yml` includes both database servers that are needed. Edit the supplied configurations in the `configurations` directory to ensure the server starts correctly. Once this is done running `docker-compose up` will start Datatools and all required database servers.
+
+## Getting in touch
+
+We have a Gitter [space](https://matrix.to/#/#transit-data-tools:gitter.im) for the full TRANSIT-Data-Tools project where you can post questions and comments.
```
Comment (miles-grant-ibigroup) [inline README.md:18]: Can we swap this link out for the gitter one?

**Review by miles-grant-ibigroup — APPROVED** [review]
Looking great pending link update

File: README.md (hunk 1 above)
Comment (philip-cline) [reply README.md:18]: The gitter one didn't actually work. I think better to just send it with the matrix link

**Review by miles-grant-ibigroup — APPROVED** [review]
New link not working either

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Link works fine for me.

**Pull request merged**